sdk/include/sdk/DataPoint.h

```diff
@@ -25,6 +25,15 @@
 
 namespace velocitas {
 
+/**
+ * @brief See
+ * https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/timestamp.proto
+ */
+struct Timestamp {
+    int64_t seconds;
+    int32_t nanos;
+};
+
```

Comment:
Later we could replace by (or offer that additionally) std::chrono::time_point ...